Resources/Prototypes/Entities/Effects/poster.yml

```
@@ -0,0 +1,15 @@
+- type: entity
+  id: EffectPosterPlacing
+  noSpawn: true
```

Test fail is real. Use `categories: [ HideSpawnMenu ]` instead

```suggestion
  categories: [ HideSpawnMenu ]
```